source/cli-translate.js

```diff
@@ -0,0 +1,128 @@
+var imagemagick_cli_lists = null;
+
+function addSelectOptions(node)
+{
+    var selectCliLanguageNode = document.createElement("select");
+    selectCliLanguageNode.className += "btn text-dark bg-light mx-4";
+    selectCliLanguageNode.style.fontSize=".75rem";
```

Comment:
Maybe better to have all these style changes as a class inside `magick-template.css`?

Reply:
Why did you resolve this conversation?

Reply:
Sorry missed this due to merges